content/miq_dialogs/miq_provision_kubevirt_dialogs_template.yaml

```diff
@@ -297,9 +297,19 @@
           :default: "1024"
           :data_type: :string
       :display: :show
+    :customize:
+      :description: Customize
+      :fields:
+        :root_password:
+          :description: Cloud User Password
+          :required: true
```

Comment:
@nasark will the clone fail if the cloud user password isn't provided?  Thinking it is possible a user could have their template configured with a root user account already created or have it set up for some e.g. LDAP system where auth is from an external provider.

Reply:
Yes, the cloud user password is used for the `cloudinitdisk` volume and is a parameter used in their templates
```
> oc get template fedora-desktop-small -o yaml
...
parameters:
- description: VM name
  from: fedora-[a-z0-9]{16}
  generate: expression
  name: NAME
- description: Name of the DataSource to clone
  name: DATA_SOURCE_NAME
  value: fedora
- description: Namespace of the DataSource
  name: DATA_SOURCE_NAMESPACE
  value: openshift-virtualization-os-images
- description: Randomized password for the cloud-init user fedora
  from: '[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}'
  generate: expression
  name: CLOUD_USER_PASSWORD
```

Reply:
https://github.com/fog/fog-kubevirt/blob/5a3ee34d92fb9abc311d5dc2f857f78d8b37566c/lib/fog/kubevirt/compute/models/template.rb#L167

Reply:
> and is a parameter used in their templates

Right but not necessarily in every template, is it possible to create a template that doesn't have that value?  I could either hard-code the password or the disk for the template might already have a user configured.

Reply:
I see, if the template is not provided by kubevirt then yes. Should we make the cloud user password `required: false` then?

Reply:
I've updated the OSV PR as well